README.md

```diff
@@ -108,10 +74,14 @@ After you're finished please remove all the comments and instructions!
 
 > Add a link to your deployed project.
 
-- [Live Demo Link](https://google.com)
+- [Live Demo Link](https://moyin.onrender.com/groups)
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
+
+## Entity Relationship Diagram <a name="er-diagram"></a>
+![er-diagram](https://github.com/microverseinc/curriculum-rails/raw/main/capstone/images/erd_diagram.png)
+
 <!-- GETTING STARTED -->
 
 ## 💻 Getting Started <a name="getting-started"></a>
```

Comment:
- [ ] Under `Setup` section:
- I would advise you to provide a proper repository to be cloned. What I pasted here is for a `Recipe_app` project